python-checks/src/test/resources/checks/use_of_empty_return_value.py

```diff
@@ -23,3 +23,9 @@ def passed_as_argument():
 
 ls = [1, 2, 3]
 result = ls.append(42)  # FN - no type inference outside of functions
+
+import win32pdh
+
+# FP due to the stubs of win32 being incomplete
```

Comment:
In this case it's not a problem, but generally it's better to keep test cases within functions, as type inference may have an influence and isn't enabled at module level.